wukong/api.py

```diff
@@ -16,6 +16,12 @@ def _add_scheme_if_not_there(url, scheme='http'):
 
     return url
 
+def _format_solr_url(url):
+    url = _add_scheme_if_not_there(url)
+    if not url.endswith('/solr/'):
```

Comment:
won't work for when host ends with `/solr` with no slash (like in our recent case 😄 )

maybe an "in" check would be better like `'/solr' in url`? or using regex?

Reply:
although, in that case, should it have a slash appended to the end?